fix-bt-a2dp.rb

```diff
@@ -64,14 +72,21 @@ def enable_a2dp(name, mac, bluez_mac, info)
 
   sleep 1
 
-  puts ">> pacmd set-card-profile bluez_card.#{bluez_mac} a2dp_sink"
-  set_card_profile_out = `pacmd set-card-profile bluez_card.#{bluez_mac} a2dp_sink`
-  unless $?.success? && set_card_profile_out == ''
-    command_failed('pacmd set-card-profile', set_card_profile_out)
-  end
+  pactl_cards = `pactl list cards`
+  command_failed('`pactl list cards`', pactl_cards) unless $?.success?
+  a2dp_profile = A2DP_PROFILES.find { |p| pactl_cards.include?(p) }
```

Comment:
It might be better to search for an exact match, for example using regex `\ba2dp_sink_ldac\b`, but I'm not all that familiar with Ruby and I thought I'd keep it simple.

Reply:
`pactl list cards` will output all cards, although we usually have only one bluetooth headphone connected in one time, I will try to improve this

Reply:
but it turns out `pactl list cards` will just output every profiles no matter the headset support or not...for example on my NW-WS623:

```
        Name: bluez_card.AA_BB_CC_DD_EE_FF
        Driver: module-bluez5-device.c
        Owner Module: 35
        Properties:
                device.description = "NW-WS623"
                device.string = "AA:BB:CC:DD:EE:FF"
                device.api = "bluez"
                device.class = "sound"
                device.bus = "bluetooth"
                device.form_factor = "headset"
                bluez.path = "/org/bluez/hci0/dev_AA_BB_CC_DD_EE_FF"
                bluez.class = "0x240404"
                bluez.alias = "NW-WS623"
                device.icon_name = "audio-headset-bluetooth"
                device.intended_roles = "phone"
        Profiles:
                headset_head_unit: Headset Head Unit (HSP/HFP) (sinks: 1, sources: 1, priority: 30, available: yes)
                a2dp_sink_sbc: High Fidelity Playback (A2DP Sink: SBC) (sinks: 1, sources: 0, priority: 40, available: no)
                a2dp_sink_aac: High Fidelity Playback (A2DP Sink: AAC) (sinks: 1, sources: 0, priority: 40, available: no)
                a2dp_sink_aptx: High Fidelity Playback (A2DP Sink: aptX) (sinks: 1, sources: 0, priority: 40, available: no)
                a2dp_sink_aptx_hd: High Fidelity Playback (A2DP Sink: aptX HD) (sinks: 1, sources: 0, priority: 40, available: no)
                a2dp_sink_ldac: High Fidelity Playback (A2DP Sink: LDAC) (sinks: 1, sources: 0, priority: 40, available: no)
                off: Off (sinks: 0, sources: 0, priority: 0, available: yes)
```